contract/src/main/java/io/neow3j/contract/Nep5.java

```diff
@@ -0,0 +1,223 @@
+package io.neow3j.contract;
+
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.methods.response.InvocationResult;
+import io.neow3j.protocol.exceptions.ErrorResponseException;
+import io.neow3j.wallet.Account;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Nep5 {
```

Comment:
Can you also add a brief Java Docs to the class level?
Very simple and short, like: "Wrapper to interact with contracts which are compatible with the NEP-5 standard."